radiant/dags/import_open_data.py

```diff
@@ -40,5 +39,15 @@
                 trigger_rule="none_failed",
             )
         )
+    data_tasks.append(
+        RadiantStarRocksOperator(
+            task_id="insert_gnomad_constraints",
+            task_display_name="gnomad_constraints Insert Data",
+            sql="./sql/open_data/gnomad_constraints_insert.sql",
+            submit_task_options=SubmitTaskOptions(max_query_timeout=3600, poll_interval=30),
+            params=ICEBERG_COMMON_TASK_PARAMS,
+            trigger_rule="none_failed",
```

Comment:
[Optional] I think those `none_failed` are not required anymore, we could remove them.